CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,4 @@ Fix: Changing cygnus port does not work (#1698)
 [cygnus-commons][MongoBackend] Proper processing of mongo errors in create index operations (#1756)
 [cygnus-ngsi] Support for NGSIv2 notifications (#953)
 [cygnus-ngsi] Changed log level for some events (#1768, #1769)
+[cygnus-ngsi][PostgisSink, PostGRESQLSink] Support for native Postgres and Postgis types (#1780)
```

Comment:
If at the end this PR has done a cleanup in the MySQLSink, maybe should it be mentioned also in CNR (in a separated line, maybe)?